### ISSUE TITLE
BCRP expression in red blood cells?

### ISSUE THREAD

**Issue by JGWojtyniak** [Open-Systems-Pharmacology/Forum#248]
Dear all,

I try to develop a model for a BCRP substrate drug and recently found the following publication: [https://www.ncbi.nlm.nih.gov/pubmed/29098941](url)
Unfortunately, unlike for metabolic enzymes, relative transporter concentration in the vascular system cannot be changed/included in the standard PK-Sim individual.  I was wondering if this is for a reason or if somebody maybe has a workaround? However, I assume the only way would be to implement this process in MoBi. If so, does anyone possibly know a good template in which something similar has been implemented that I could use as a guide?

Many thanks in advance,
Jan

**Comment by PavelBal** [comment]
I do not see a way to implement it in PK-Sim.

In MoBi, you have to "express" the transporter in the RBC. For this, create a simple PK-Sim model with any protein expressed in RBC (as you can do for enzymes). This will be your template.

In your model with BCRP, in the molecule start values - building block, select the check-box "Is Present" for BCRP in all BloodCells-containers (do not forget the mucosa-tissues!). In the parameter start values - building block, change the values of "Relative expression in blood cells" and "relative expression in blood cells (normalized)" to the value according to your calculated expression values. Then, for all "BloodCells"-compartments, set the values of the parameters "Relative expression", "Relative expression (normalized)", and "Relative expression out." to the same formulas as the RBC-expressed protein from the template model you created in PK-Sim.

Finally, you have to create a new transport process in the molecules-buidling block. Expand all molecules that are substrates of this transporter, find BCRP, right-click on it and select "Create transport". You then have to adjust the "Properties" (from RBC to plasma) and define a new reaction rate formula.

**Comment by JGWojtyniak** [comment]
@PavelBal: Thank you very much for the detailed explanation! I recently found the time to test it and it worked like a charm. However, let me add a few experiences in case someone want to reproduce the method:

1. Since "relative expression in blood cells" and "relative expression in blood cells (normalized)" is not a parameter for transporters in the "Molecules" building block I had first to add the parameter. Afterwards it was sufficient to hit the "Extend" button in the "molcecule start values" building block.

2. Apart from the mucosa-tissue it is important to not forget the pericentral and periportal liver compartments!

3. With Pavel's guidance we added a efflux-transport process which I will upload as MoBi pkml file.

4. Additionally, we created an observer to fit the blood plasma ratio (which was the basic idea why I wanted to implement the process). I put the observer also in the attached .zip file.

Hope it helps! If someone has a question about this procedure feel free to ask.

Here are the files: [building_blocks.zip](https://github.com/Open-Systems-Pharmacology/Forum/files/2661966/building_blocks.zip)

PS: I think this issue can be closed now.
